src/resources/views/crud/inc/form_save_buttons.blade.php

```diff
@@ -22,3 +22,27 @@
 
     <a href="{{ $crud->hasAccess('list') ? url($crud->route) : url()->previous() }}" class="btn btn-default"><span class="fa fa-ban"></span> &nbsp;{{ trans('backpack::crud.cancel') }}</a>
 </div>
+
+@push('after_scripts')
+<script>
+    
+    // make Save Buttons that are anchors behave like Submit buttons (trigger HTML5 validation)
+    jQuery(document).ready(function($) {
+        var selector = $('#bpSaveButtonsGroup').next();
+        $(selector).find('a').each(function() {
+            $(this).click(function(e) {
+                e.stopPropagation()
+                let form = $(this).closest('form');
+                if (!form[0].checkValidity || form[0].checkValidity()) {
+                    form.submit();
```

Comment:
@pxpm I'm not sure this line is the problem. But there is problem 😄 My tests show that this PR breaks the functionality for the "Save and new" and "Save and edit" buttons.

No matter what Save button you click, the functionality that gets triggered is "Save and back".

![2020-01-03 10 11 53](https://user-images.githubusercontent.com/1032474/71713262-a83b4600-2e11-11ea-9977-9ab39059429f.gif)

Let me know if I haven't explained myself correctly, I'm not sure I have.

Cheers!

Reply:
Totally right @tabacitu ! Forgot to update the saveAction in the input, my bad! Guess I found the code now in [form_content.blade.php](https://github.com/Laravel-Backpack/CRUD/blob/27c5d9dd2d01b8b3ada5ba29cc587c5e546f5959/src/resources/views/crud/form_content.blade.php#L59)

Should I add this code to complete the one in `form_content` or remove the bits from there and have it in this file ?

Best,
Pedro

Reply:
Makes perfect sense to move the Save button logic here, from ```form_content```. Totally agree!